apps/factory/src/pages/dashboard/prompts/[id]/index.tsx

```diff
@@ -160,12 +161,29 @@ const PackageShow: NextPageWithLayout = () => {
             ) : (
               <span></span>
             )}
-            <CreateTemplate
-              pp={pp as pp}
-              onCreate={ptCreateMutation.mutate}
-              status={status}
-              customError={customError}
-            ></CreateTemplate>
+            <Grid component="span">
+              <Grid container>
+                <Grid item xs={6} md={6} lg={6}>
+                  <CreateTemplate
+                    pp={pp as pp}
+                    onCreate={ptCreateMutation.mutate}
+                    status={status}
+                    customError={customError}
+                    ptId={""}
+                  ></CreateTemplate>
+                </Grid>
+                <Grid item xs={6} md={6} lg={6}>
+                  <CreateTemplate
+                    pp={pp as pp}
+                    onCreate={ptCreateMutation.mutate}
+                    status={status}
+                    customError={customError}
+                    ptId={ptId}
+                  ></CreateTemplate>
```

Comment:
please use as below
<A ptid=false> => create new, 
{ptid && (<A ptid={ptid}>} => for edit